### PR TITLE
Fix connector property loading issue.

### PR DESCRIPTION
**Pull request by wijith7**

### CODE REVIEW

**Comment by JKAUSHALYA** [comment]
Fixes https://github.com/wso2/product-is/issues/4928

File: components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityGovernanceServiceImpl.java [hunk 1]
```diff
@@ -120,6 +123,38 @@ public void updateConfiguration(String tenantDomain, Map<String, String> configu
...
+                    newProperty.setDisplayName(identityConnectorConfig.getPropertyNameMapping().get(connectorProperty));
+                    Properties defaultPropertyValues = identityConnectorConfig.getDefaultPropertyValues(tenantDomain);
+                    newProperty.setValue(String.valueOf(defaultPropertyValues.get(connectorProperty)));
+                    updateConfiguration(tenantDomain, Collections.singletonMap(newProperty.getName(), newProperty.getValue()));
+                    propertiesToAdd.add(newProperty);
+                }
+            }
+        }
+
+        if (propertiesToAdd.size() > 0) {
```
Comment (pulasthi7) [inline components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityGovernanceServiceImpl.java]: Use the isEmpty check instead

File: components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityGovernanceServiceImpl.java [hunk 2]
```diff
@@ -120,6 +123,38 @@ public void updateConfiguration(String tenantDomain, Map<String, String> configu
             configMap[index] = property;
             index++;
         }
+
+        List<IdentityConnectorConfig> identityConnectorConfigs = getConnectorList();
+        String[] connectorProperties;
+
+        ArrayList<Property> propertiesToAdd = new ArrayList<>();
```
Comment (pulasthi7) [inline components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityGovernanceServiceImpl.java]: Better to store the reference as List

Comment (wijith7) [reply components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityGovernanceServiceImpl.java]: fixed by another commit

File: components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityGovernanceServiceImpl.java (hunk 1 above)
Comment (wijith7) [reply components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityGovernanceServiceImpl.java]: fixed by another commit

**Pull request merged**